other-tabs/collaboration/distribute-tasks-with-inboxes.md

```diff
@@ -0,0 +1,97 @@
+{% hint style="info" %}
+You must [contact a CS](https://www.forestadmin.com/contact-us?message=Subject:%20I%20want%20to%20enable%20the%20inbox%20feature) to have access to this feature.
+{% endhint %}
+
+The Inbox feature automatically assigns tasks among your team members based, improving the efficiency of your workflows.
+
+## Creating an inbox
+
+Once the feature is enabled for your project, navigate to the Collaboration tab and enable the Edit Layout mode.
+
+In the left menu, find the "Inboxes" section and click "Create Inbox"
+
+An inbox is always based on a segment of a collection. In this example, we will create an inbox based on the `Documents` collection and the `Waiting for Validation` segment, which contains all the documents that have not been verified yet.
+
+You can then select between two dispatch rules:
+
+- "Field based":
+  - You can select up to 3 fields.
+  - For each field, you can sort ascending or descending.
+  - The records will be sorted by the first sorting field, in case of equality, the second sorting field is applied and so on to the third sorting field.
+  - For example:
+    - You can select the first sorting field to be the create date of the document, sorted descending
+    - You can select the second sorting field to be the importance of the document, sorted ascending
+    - For two `Documents` with the same creation date, the importance will take over for the first that will be assigned
+- "Random":
+  - The `Documents` will be assigned in a random order
+
+## Processing tickets
+
+Once an inbox has been created, access the inbox from the "Collaboration" tab.
+
+### Todo section
+
+The count near the "Todo" section represents the number of records currently in the collection segment selected for the inbox (e.g. `Waiting for Validation`) that have not yet been assigned to an operator.
+
+In the "Todo" section of the inbox, an operator can click on the "Start processing button" to have a task automatically assigned, and being redirected to the record.
+
+The operator then does what is necessary to achieve the current workflow, for example trigger a `Validate Document` action. Then the operator can click on the "Next ticket" button on the top of Forest Admin UI, to be assigned to a new record.
+
+### Doing section
+
+On the "Doing" section, the operator can view the tasks that are still in the segment and that are currently assigned to he or she.
+
+### Backlog section
+
+This section is accessible for operators with a Manager, Editor or Administrator permission level.
+
+It provides an overview of the current inbox.
+
+You can see all the records in the segment with their current task state, and which operator is assigned to it.
+
+For the "Doing" tasks, you can see the date when the task was started.
+
+For the "Completed" tasks, you can also see the time it took for the task to get out of the segment. If the records gets out of the segment using an action requiring an approval, it includes the approval time.
```

Comment:
`to get out of the segment` is not really true. handling time seems more correct